packages/influxdb_exporter/packaging

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eux
+
+# Copy common utils
+mkdir -p ${BOSH_INSTALL_TARGET}/common
+cp -a ${BOSH_COMPILE_TARGET}/common/* ${BOSH_INSTALL_TARGET}/common
+
+# Build influxdb_exporter package
+export GOROOT=/var/vcap/packages/golang_prometheus/go
+export PATH=${GOROOT}/bin:${PATH}
+
+SOURCE_DIR=github.com/prometheus/influxdb_exporter
+PACKAGE_NAME=prometheus/influxdb_exporter
```

Comment:
This may work now, but in the future, it will lead us to problems. Better to remove the `SOURCE_DIR` and use just the `PACKAGE_NAME` like:

```
PACKAGE_NAME=github.com/prometheus/influxdb_exporter
mkdir -p ${BOSH_INSTALL_TARGET}/src/${PACKAGE_NAME}
cp -a ${BOSH_COMPILE_TARGET}/${PACKAGE_NAME}/* ${BOSH_INSTALL_TARGET}/src/${PACKAGE_NAME}
export GOPATH=${BOSH_INSTALL_TARGET}
```

The reason why I'm asking to change this is that the namespace is `github.com/prometheus/influxdb_exporter`. By changing the `PACKAGE_NAME` to a different namespace (`prometheus/influxdb_exporter`), if someday the `influxdb_exporter` imports some source code in the same namespace (ie. a dir inside the main repo), then the build will fail, because it will be unable to find the imports.

The trick to use a different `PACKAGE_NAME` works at the [kube_state_metrics_exporter package](https://github.com/cloudfoundry-community/prometheus-boshrelease/blob/master/packages/kube_state_metrics_exporter/packaging) because they use a different namespace.